examples/dash/dash-customization/README.md

```diff
@@ -0,0 +1,131 @@
+# Comprehensive Guide to Dash UI Customization
+
+Explore various methods to customize your Dash app UI, from using styling with CSS and Dash Bootstrap Components, to building custom Dash components to achieve advanced customization using React.
+
+1. Inline CSS Styling
+2. Using Your Own CSS Files
+3. Dash Bootstrap Components 
+4. Custom Callbacks for Dynamic Styling
+5. Building Custom Dash Components
+
+Each section includes simple scripts that feature a title and a button, demonstrating how these methods can be practically applied to enhance your application's aesthetics and functionality.
+
+## Methods for UI Customization in Dash
+### 1. Inline CSS Styling
+#### Example Code
+- `app1.py`
```

Comment:
It would be better to use more meaningful names for files , e.g., `app_inline.py`, `app_custom.py`, etc

Reply:
I agree, make sure to update the blog as well with these names

Reply:
app1 changed to `app_inline.py` [here](https://github.com/ploomber/doc/pull/273/files#diff-a449c1cbe428696ea7f25cddd2150ca0712824cfa2acdf5736691e1c5f8abf4eR16)

app1 changed to `app_inline.py` [here](https://github.com/ploomber/doc/pull/273/files#diff-a449c1cbe428696ea7f25cddd2150ca0712824cfa2acdf5736691e1c5f8abf4eR16)

app1 changed to `app_inline.py` [here](https://github.com/ploomber/doc/pull/273/files#diff-a449c1cbe428696ea7f25cddd2150ca0712824cfa2acdf5736691e1c5f8abf4eR16)

app1 changed to `app_inline.py` [here](https://github.com/ploomber/doc/pull/273/files#diff-a449c1cbe428696ea7f25cddd2150ca0712824cfa2acdf5736691e1c5f8abf4eR16)